contexts-config-store/store/contexts_config_store.go

```diff
@@ -37,3 +38,18 @@ func GetContextsConfigStore() ContextsConfigStore {
 	})
 	return contextsConfigStore
 }
+
+func IsRemote(kurtosisContext *generated.KurtosisContext) bool {
+	var isRemote bool
+	_, _ = golang.Visit[struct{}](kurtosisContext, golang.KurtosisContextVisitor[struct{}]{
```

Comment:
the visitor pattern has been quite useful so far; i think I am seeing the 5th implementation of this for remote/local haha

Reply:
visitors are the best :D